unstructured/ingest/doc_processor/generalized.py

```diff
@@ -59,9 +60,11 @@ def process_document(ingest_doc_json: str, **partition_kwargs) -> Optional[List[
         # the results. Instead, the Processor (caller) may work with the aggregate
         # results across all docs in memory.
         doc.write_result()
-    except Exception:
+    except Exception as e:
         # TODO(crag) save the exception instead of print?
         logger.error(f"Failed to process {doc}", exc_info=True)
+        raise e
```

Comment:
if we raise here, does this kill other subprocesses from completing?

Reply:
That’s outdated. Had it in for debugging. Should be gone now.